components/script/dom/htmlanchorelement.rs

```diff
@@ -544,7 +544,11 @@ impl Activatable for HTMLAnchorElement {
         if let Some(element) = target.downcast::<Element>() {
             if target.is::<HTMLImageElement>() && element.has_attribute(&local_name!("ismap")) {
                 let target_node = element.upcast::<Node>();
-                let rect = target_node.bounding_content_box();
+                // TODO(emilio): Should we assert that the element is rendered
+                // (unwrap the content box)?
```

Comment:
Pretty sure this could break in the present of `img.click()` when it's not rendered.